recipes/treeshrink/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.3.9" %}
+
+package:
+  name: treeshrink
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/uym2/TreeShrink/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 460894699250b7fc1a7096d84bebb0b52b5551612e9faa283c600a15d3f756a3
+
+requirements:
+  host:
+    - python >=3.6,<=3.9.15
```

Comment:
This pin looks strange. Is that really needed?

Reply:
Yes, at least until treeshrink is updated upstream; it makes use of `collections.MutableMapping` which is unavailable in Python >=3.10.